### PR TITLE
Adding name parameter to recipe

### PR DESCRIPTION
**Pull request by prakashmrec**

### CODE REVIEW

File: modules/core/src/test/java/com/smartbear/readyapi4j/TestRecipeBuilderTest.java [hunk 1]
```diff
@@ -59,6 +59,13 @@ public void buildRecipeWithTestCaseProperty() {
     }
 
     @Test
+    public void buildsRecipeWithName() throws Exception {
+        String recipeName = "AuthenticationRecipe";
+        TestRecipe testRecipe = newTestRecipe().named(recipeName).buildTestRecipe();
+        assertThat(testRecipe.getTestCase().getName(), is(recipeName));
```
Comment (olensmar) [inline modules/core/src/test/java/com/smartbear/readyapi4j/TestRecipeBuilderTest.java]: can we expose the name with testRecipe.getName() also? it's a bit confusing for someone new to this model that you have to get the name from the testCase when you set it on the recipe!?

Comment (prakashmrec) [inline modules/core/src/test/java/com/smartbear/readyapi4j/TestRecipeBuilderTest.java]: Sure, I will fix.

Comment (prakashmrec) [inline modules/core/src/test/java/com/smartbear/readyapi4j/TestRecipeBuilderTest.java]: Fixed.

**Comment by olensmar** [comment]
thanks!

**Pull request merged**